web/frontend/package.json

```diff
@@ -17,7 +17,7 @@
     "@improbable-eng/grpc-web": "~0.15.*",
     "@viamrobotics/prime": "~0.2.*",
     "@viamrobotics/rpc": "~0.1.*",
-    "@viamrobotics/sdk": "0.3.0-rc.0",
+    "@viamrobotics/sdk": "^0.3.1-next.0",
```

Comment:
For peer deps we want to use this syntax for resolution: https://github.com/viamrobotics/rdk/pull/2754/files#diff-0816cd46d529b043bd6c753c560b74405e043bc40c7b3f9fc9083d16ef23ed46R17

Reply:
I don't understand which syntax you're referring to since `@viamrobotics/sdk` was not changed in that PR. Are you saying it should be `">=0.3.1-next.0"` or `"0.3.1-next.0"`?

Reply:
For now we should probably lock the version since we're using `next`, so `0.3.1-next.0`. When we feel confident enough that patch upgrades won't affect much other than bugfixes, we can do this: `>=0.3`. This will mean that consumers of this library (app.viam.com) can have mismatches in patch version without any resolution issues, the latest patch will be used for both app and RC.